doc/contribute/project_roles.rst

```diff
@@ -0,0 +1,279 @@
+  * May have a maintainer, shall have a higher-level platform maintainer
+* Drivers
+
+  * Shall have a driver-area (and API) maintainer
+  * Could have individual driver implementation maintainers but preferably collaborator/contributors
+  * In the above case, platform-specific PRs may be re-assigned to respective collaborator/contributor of driver implementation
+
+
+Merge Criteria
+##############
+
+* All continuous integration checks have passed
+
+  * Codeowners
+  * Device Tree
+  * Documentation
+  * Gitlint
+  * Identity/Emails
+  * Kconfig
+  * License
+  * Checkpatch (Coding Style)
+  * Pylint
+  * Sanitycheck + Other Unit tests
+  * Simulated Bluetooth Tests
+
+* Planned
+
+  * Footprint
+  * Code coverage
+  * Coding Guidelines
+  * Static Analysis (Coverity)
+  * Documentation coverage (APIs)
+
+* PR template with checklist
+
+* Minimal of 2 approvals
+
+  * A collaborator from the same subsystem.
+  * Alternately another maintainer of another subsystem
+  * Approval by the assignee
+
+* A minimum review period of 2 days, 4 hours for trivial changes (see
+  :ref:`review_time`). Hotfixes can be merged at any time after CI passes.
+* All required checks are passing
+
+Escalation Process
+##################
+
+* Contributors may object to change requests or decisions made by Maintainers.
+* Process
+
+  * Resolve in the PR among assignee, maintainers and reviewer
+
+    * Assignee to act as moderator if applicable
+
+  * Optionally resolve in the dev review meeting with more maintainers and project stakeholders
+
+    * The involved parties and the Assignee to be present when the (escalated) issue is discussed
+
+  * TSC: Assignees can escalate to the TSC voting members and get a binding resolution in the TSC.
+  * Assignee to ensure the resolution of the escalation is reflected in the PR review.
+
+
+Release Manager
+###############
+
+A *Maintainer* responsible for driving a specific release to
+completion following the milestones and the roadmap of the
+project for this specific release.
+
+* TSC has to approve a release manager.
+
+A Release Manager joins the release engineering team and has
+the rights and responsibilities of that team in addition to
+the following:
+
+* Right to manage and coordinate all code merges after the
+  code freeze milestone (M3, see `program management overview <https://wiki.zephyrproject.org/Program-Management>`_.)
+* Responsibility to drive and coordinate the triaging process
+  for the release
+* Responsibility to create the release notes of the release
+* Responsibility to notify all stakeholders of the project,
+  including the community at large about the status of the
+  release in a timely manner.
+* Responsibility to coordinate with QA and validation and
+  verify changes either directly or through QA before major
+  changes and major milestones.
+
+Roles / Permissions
+###################
+
+.. table:: Project Roles vs Github Permissions
```

Comment:
I strongly recommend switching to the other table format:

```
.. list-table::
   :header-rows: 1

   * - ..
     - ..
     - **admin**

   * - Main Roles
     - Contributor
     - 

```

```

Reply:
@carlescufi Zephyr Documentation accepts this table format, check here:

https://docs.zephyrproject.org/latest/guides/documentation/index.html

and the given link:
https://docutils.sourceforge.io/docs/ref/rst/restructuredtext.html#grid-tables

Actually, the current table format is the best option to get all this complex merged rows and columns